cypress/integration-qe/basicUserAuthentication.spec.js

```diff
@@ -0,0 +1,18 @@
+import { verifyPullSecretHosted } from './ocmShared';
+
+describe('test basic user authentication', () => {
+  it('navigate from openshift portal to assisted installer', () => {
+    cy.visit('');
+    cy.get('[aria-current="page"]').contains('Clusters').click();
+    cy.get('button').contains('Create cluster').click();
```

Comment:
nit: to make it robust, let's do not assume that there is just single `<button>` on the page